modules/ossm-jaeger-config-elasticsearch.adoc

```diff
@@ -0,0 +1,43 @@
+////
+This TASK module included in the following assemblies:
+-ossm-configuring-jaeger.adoc
+////
+
+[id="ossm-jaeger-config-elasticsearch_{context}"]
+= Configuring the Elasticsearch index cleaner job
+
+When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` it also creates the custom resource definition (CRD) for Jaeger.  The Jaeger operator then uses this CRD when creating Jaeger instances.
+
+When using Elasticsearch storage, by default a job is created to clean old traces from it.  To configure the options for this job, you edit the Jaeger custom resource (CR) are listed below so you can customize it for your use case.
```

Comment:
"you edit  the Jaeger custom resource (CR), to customize it for your use case. The relevant options are listed below."?